DOL.WHD.Section14c.Web/src/modules/components/filters/dateFilter.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = function(ngModule) {
+  ngModule.filter('dateFilter', function(moment) {
+    return function(input) {
+      // parse to moment object
+      let outVal = 'INVALID DATE';
+      const dateValMoment = moment(input, moment.ISO_8601, true);
+      if (dateValMoment.isValid()) {
+        // parse out values
+        const month = dateValMoment.month() + 1; // month is zero-based
+        const day = dateValMoment.date();
+        const year = dateValMoment.year();
+        outVal = `${month}/${day}/${year}`;
+      }
+
+      return outVal;
+    };
+  });
+};
```

Comment:
This exact code already exists in `src/modules/filters/dateFilter.js`.  Why does it need to be here too?  Is there somewhere else this can be put so that it can be shared instead of duplicating it?

Reply:
I like how you deleted this before I finished my review.  😂